helm_deploy/values-prod.yaml

```diff
@@ -9,7 +9,9 @@ generic-service:
   env:
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.json
     OAUTH_ENDPOINT_URL: "https://sign-in.hmpps.service.justice.gov.uk/auth"
+    PRISON_API_URL: "https://api.prison.service.justice.gov.uk"
     BASM_ENDPOINT_URL: "https://api.bookasecuremove.service.justice.gov.uk"
+    PRISONER d_SEARCH_API_ENDPOINT_URL: "https://prisoner-offender-search.prison.service.justice.gov.uk"
```

Comment:
I just happened to be passing but this looks like a typo for your search endpoint env ??